utils/R/read.output.R

```diff
@@ -178,6 +178,7 @@ read.output <- function(runid, outdir, start.year = NA, end.year = NA, variables
       for (i in seq_along(n)) {
         y <- c(y, rep(years[i], n[i]))
       }
+      model$year<-rep(NA, seq_along(n))
```

Comment:
Will this change have any effect? I would expect line 182 to promptly clobber this one.